state/test/withState.ts

```diff
@@ -142,7 +142,7 @@ describe('withState', function() {
     function Parent(sources: ParentSources): ParentSinks {
       const childSinks = isolate(Child, {state: 'child'})(sources);
       return {
-        state: childSinks.state,
+        state: childSinks.state as Stream<Reducer<Parent>>,
```

Comment:
This case is a true example of an inference that we don't yet know how to support. Previously Isolate was inferring these cases as either `Stream<Reducer<any>>` or `Stream<Reducer<{}>>`, with this PR, the type is `Stream<unknown>` so it breaks on purpose unless the developer does a type cast, like we're doing here.